app/models/user.rb

```diff
@@ -30,6 +35,10 @@ class User < ApplicationRecord
   has_many    :chat_channels, through: :chat_channel_memberships
   has_many    :push_notification_subscriptions, dependent: :destroy
   has_many    :feedback_messages
+  has_many :in_mentor_relationships, class_name: "MentorRelationship", foreign_key: "mentee_id"
```

Comment:
"in_mentor_relationships" and "out_" are pretty kind of confusing words for me. Doesn't seem to say what's happening. Is there a technical reason for this wording?